tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckQuotes.java

```diff
@@ -10,6 +10,8 @@
 import com.ibm.icu.text.UnicodeSet;
 
 public class CheckQuotes extends CheckCLDR {
+    private static final String VALID_DELIMITER_URL = "https://cldr.unicode.org/translation/characters";
```

Comment:
So this page was originally at: https://cldr.unicode.org/translation/characters-emoji-symbols 

And had 3 subpages:

https://cldr.unicode.org/translation/characters/character-labels
https://cldr.unicode.org/translation/characters/emoji-names-and-keywords -> parameter would need to be updated to align with original.
https://cldr.unicode.org/translation/characters/typographic-names

I am happy to revert to match the old site, but I'd like to confirm that you agree on the approach before making the changes.

Reply:
Will update the site to link the subpages on the main header page which didn't migrate properly.

Reply:
Per today's meeting I'll change all remaining "characters-emoji-symbols" to "characters"

Reply:
Actually do you think this was the original characters page? https://cldr.unicode.org/translation/core-data/alphabetic-information

I am wondering after looking up some of the other links you weren't able to find originally.